subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/plugins/JacocoPlugin.java

```diff
@@ -179,7 +179,7 @@ public void execute(JacocoReport reportTask) {
     }
 
     private void configureJacocoReportDefaults(final JacocoPluginExtension extension, final JacocoReport reportTask) {
-        reportTask.getReports().all(new Action<ConfigurableReport>() {
+        reportTask.getReports().withType(ConfigurableReport.class, new Action<ConfigurableReport>() {
```

Comment:
there's not really any point doing this if we're going to change `JacocoReportsContainer` to contain `ConfigurableReport` rather than `Report`. It's not wrong, it's just not necessary and does a little more work than just using `all()`.